src/components/KonnectorTile.jsx

```diff
@@ -70,33 +43,45 @@ export class KonnectorTile extends Component {
       t
     } = this.props
     const { domain, secure } = this.context
-    const hideKonnectorErrors = flag('hide_konnector_errors')
+
+    let status
+
+    if (konnector.available_version) status = STATUS.UPDATE
+    else if (isInMaintenance) status = STATUS.MAINTENANCE
+    else if (error || userError) status = STATUS.ERROR
+    else if (!accountsCount) status = STATUS.NO_ACCOUNT
+    else status = STATUS.OK
+
     return (
       <NavLink
-        className="item-wrapper"
+        className={classNames('item-wrapper', {
+          '--ghost': status === STATUS.NO_ACCOUNT,
+          '--maintenance': status === STATUS.MAINTENANCE
+        })}
         to={route}
         title={getKonnectorError({ error, t })}
       >
-        <div
-          className={classNames(
-            'item-icon',
-            getErrorClass({
-              accountsCount,
-              error,
-              hide: hideKonnectorErrors,
-              userError,
-              hasUpdate: !!konnector.available_version,
-              isInMaintenance
-            })
-          )}
-        >
+        <div className="item-icon">
           <AppIcon
             alt={t('app.logo.alt', { name: konnector.name })}
             app={konnector}
             domain={domain}
             secure={secure}
           />
-          <Icon icon={brokenIcon} className="konnector-state" />
+          {status === STATUS.MAINTENANCE && (
+            <Icon
+              icon="wrench-circle"
+              className="item-status"
+              color={palette.coolGrey}
+            />
+          )}
+          {status === STATUS.ERROR && (
+            <Icon
+              icon="warning-circle"
+              className="item-status"
+              color={palette.pomegranate}
+            />
+          )}
```

Comment:
Why not having something more declarative in our config like: 

const STATUS = [{
ERROR : {
icon: "warning-circle",
color: pomegranate
}...
}]

I find it easier to read but also easier to maintain and update (and override)